src/android/app/src/main/java/org/citra/citra_emu/overlay/InputOverlay.kt

```diff
@@ -110,6 +110,11 @@ class InputOverlay(context: Context?, attrs: AttributeSet?) : SurfaceView(contex
                 TurboHelper.setTurboEnabled((!TurboHelper.isTurboSpeedEnabled()))
             }
 
+            if (button.id == NativeLibrary.ButtonType.BUTTON_COMBO) {
+                NativeLibrary.onGamePadEvent(NativeLibrary.TouchScreenDevice, NativeLibrary.ButtonType.BUTTON_A, button.status)
+                NativeLibrary.onGamePadEvent(NativeLibrary.TouchScreenDevice, NativeLibrary.ButtonType.BUTTON_X, button.status)
+            }
```

Comment:
This code kind of speaks for itself.
I'll close this PR, but I'm open to an improved implementation